samples/boards/nrf/nrf53_sync_rtc/net/src/main.c

```diff
@@ -62,10 +62,9 @@ static int mbox_init(void)
 
 static int ipm_init(void)
 {
-	const struct device *ipm_dev;
+	const struct device *ipm_dev = DEVICE_DT_GET_ONE(nordic_mbox_nrf_ipc);
```

Comment:
I don't think this is correct. `nordic,mbox_nrf_ipc` is an MBOX device but you are trying to use an IPM API on that.